### ISSUE TITLE
Stock OVMF is unable to start OpenCore

### ISSUE THREAD

**Issue by hexchain** [Leoyzen/KVM-Opencore#8]
I'm trying to set up KVM-OSX on Arch Linux, but the stock OVMF from the [`edk2-ovmf`](https://www.archlinux.org/packages/extra/any/edk2-ovmf/) package does not work:

![image](https://user-images.githubusercontent.com/529520/83951764-2e7e0d00-a834-11ea-88a1-fb1b451ac59b.png)

I've tried to use both the `OpenCore.qcow2` from the repository and a newly built one using the script, but neither worked.

Please let me know if more information (e.g. EDK2 debugging logs) is needed.

**Comment by dslunjski** [comment]
I was 100% sure I messed up somewhere ... but now seeing this. 
I get the exact same Access Denied messages.
Also running up to date Arch and using stock edk2-ovmf.

edk2-ovmf 202002-9
qemu 5.0.0-7
linux-lts 5.4.41-1

I'm using Clover for now because I couldn't get around this.

**Comment by Acelogic** [comment]
Yeah typically the stock OVMF doesn't work for me. I always have to stick to the patched one from the simple-MacOS-KVM repo
